xa30_workaround/scripts/dcmdat2niix.py

```diff
@@ -145,9 +207,16 @@ def main():
                 # TODO: remove later if fixed
                 # resave the phase image with the dat data
                 if "_ph" in nifti.name:
-                    nib.Nifti1Image(data_array[..., 0, :], nifti_img.affine, nifti_img.header).to_filename(
-                        nifti_img_path
-                    )
+                    if len(shape) <= 3:
```

Comment:
Geez, I can't believe I write this for loop iterating over the `TEs`... awful nesting.

I think this is all fine, probably needs a refactor by moving to a separate function so it's tidier.